### PR TITLE
Add Zahlen Raten

### PR DESCRIPTION
**Pull request by Koch-Dustin**
Changes:
   - Add Zahlen Raten

PR: #01

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import random
+    
+def game():
+  numberToGuess = random.randint(1, 100) 
+  trysTakeToGuess = 0
+
+  print("Ich denke an eine Zahl zwischen 1 - 20. Kannst du sie erraten?")
```
Comment (DevBastiS) [inline main.py:7]: That is a lie ;)

File: main.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  userGuessedNumber = -1
```
Comment (DevBastiS) [inline main.py:9]: Why not 0?

File: main.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        print("Sehr Warm")
+      else:
+        print("Klasse! Du hast meine Zahl erraten!")
+        print("Gebrauchte Versuche: " + str(trysTakeToGuess))
+        playAgain = input("Willst du nochmal spielen? [Y / N] : ")
+        if(playAgain == "Y"):
+          game()
+        break
+    
+    else:
```
Comment (DevBastiS) [inline main.py:45]: You can use early exit here ;)

**Review by DevBastiS — APPROVED** [review]

**Review by DevBastiS — APPROVED** [review]

File: main.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import random
+    
+def game():
+  numberToGuess = random.randint(1, 100) 
+  trysTakeToGuess = 0
```
Comment (MKrips) [inline main.py:5]:
```suggestion
  triesNeededToGuess = 0
```

**Review by MKrips — CHANGES_REQUESTED** [review]

File: main.py (hunk 3 above)
Comment (Koch-Dustin) [reply main.py]: When i implement "break" it prints out the wanted message and then it stops running the file

**Review by MKrips — APPROVED** [review]

**Pull request merged**